styles.css

```diff
@@ -204,6 +262,25 @@ nav a:hover {
     border-radius: 25px;
 }
 
+@media screen and (max-width: 768px) {
+    .portfolio-item-row {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        width: 90%;
+    }
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
This can be combined with the media query above, as they both have the same max-width.